src/scripts/main.js

```diff
@@ -22,6 +22,20 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  for (const key in data) {
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    ul.append(li);
```

Comment:
<img width="1085" alt="image" src="https://github.com/mate-academy/js_tree-from-object-DOM/assets/70220667/1559e74f-6306-4129-b8df-195c65d82745">
why do you need to create the ul for each item

Reply:
create it one time before the loop

Reply:
I realized and corrected, thanks.